benchmark/error/node-error.js

```diff
@@ -3,19 +3,21 @@
 const common = require('../common');
 
 const bench = common.createBenchmark(main, {
-  n: [1e7],
+  n: [1e6],
+  code: ['built-in', 'ERR_HTTP2_STREAM_SELF_DEPENDENCY'],
```

Comment:
```suggestion
  code: ['built-in', 'ERR_INVALID_STATE'],
```

why it has changed?

Reply:
Because I wanted a simple Error, which can be instantiated without any parameters. ERR_INVALID_STATE needs parameters. I arbitrary chose it.
I wanted to measure the performance of the instantiation, not the performance when the errors have parameters and need to create a string, or when message is a function.

Reply:
When we check a PR that's written for performance, the benchmark are run to verify all important use cases it can affect, not just the use cases that it tries to optimize. If it speeds up X but slows down Y, Y should be benchmarked before the PR lands, not after.

Reply:
added benchmarks. PTAL